metadata-models/src/main/pegasus/com/linkedin/schema/SchemaField.pdl

```diff
@@ -41,4 +42,9 @@ record SchemaField {
    * There are use cases when a field in type B references type A. A field in A references field of type B. In such cases, we will mark the first field as recursive.
    */
   recursive: boolean = false
+
+  /**
+   * Tags associated with the field
+   */
+  globalTags: optional GlobalTags
```

Comment:
Same concern as what I left above. It doesn't make sense for me if "globalTags" property has to be added into any "taggable" things.

Reply:
I think it would be nice if pdl supported inheritance... but it doesn't .. so the only way to add in fields is to either manually write it ... or have it be "included" in.